projects/vendor/src/yotpo/yotporeview/yotporeview.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, ElementRef } from '@angular/core';
+import { BaseyotpoComponent } from './../baseyotpo/baseyotpo.component';
+import { ProductService, RoutingService } from '@spartacus/core';
+
+@Component({
+  selector: 'cx-yotporeview',
+  templateUrl: './yotporeview.component.html',
+  styleUrls: ['./yotporeview.component.css']
```

Comment:
The files should be `.scss` but for both components we don't have any style you can just delete both css files.

Reply:
fixed